src/core/server/saved_objects/import/import_saved_objects.ts

```diff
@@ -60,11 +60,12 @@ export async function importSavedObjectsFromStream({
   dataSourceId,
   dataSourceTitle,
   workspaces,
-  dataSourceEnabled,
 }: SavedObjectsImportOptions): Promise<SavedObjectsImportResponse> {
   let errorAccumulator: SavedObjectsImportError[] = [];
   const supportedTypes = typeRegistry.getImportableAndExportableTypes().map((type) => type.name);
 
+  const dataSourceEnabled = supportedTypes.includes('data-source');
```

Comment:
can we get it from config? Not sure if I understand what we try to achieve here

Reply:
typeRegistry is from the config, this is the savedObjectConfig: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/server/saved_objects/saved_objects_service.ts#L372

Reply:
We are checking a config not a supported type, right?

Reply:
yeah, the supported type is from config

Reply:
How is supported type configured? How is it related to if data source is enabled config?

Reply:
yeah, when constructing the saved object client, it will read config from the raw YAML file, and register it, if data source is enabled, it will support data-source type

Reply:
So the existing meaning of `dataSourceEnabled` was that it's a supported type, and now we're just reading it directly?